app/workers/classifications_dump_worker.rb

```diff
@@ -1,7 +1,10 @@
 require 'csv'
 
 class ClassificationsDumpWorker
+  include Sidekiq::Worker
   include DumpWorker
+  include DumpMailerWorker
```

Comment:
Sorry I still don't get why this would be included in the non-Mailer Workers. Shouldn't the `DumpWorker` module contain all the necessary functionality for a data dump worker? And the `DumpMailerWorker` module should only have common functionality for the data dump mailer workers?